package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "nodejs-st-22-rest-api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
```

Comment:
you probably should add yourself as an author of the work, if you are, actually, the one...

Reply:
Done.